Makefile.am

```diff
@@ -92,6 +90,7 @@ rpm: local
 
 release: tag
 	$(MAKE) archive
+	$(MAKE) -C po/ $(PACKAGE_NAME).pot-update
```

Comment:
Oh, I see. The .pot update can now go from ``bumpver`` as the file is not tracked by git either. However, have you tried if the .pot file pushed to zanata has the right version set? I had to add ``./autogen.sh && ./configure`` before generating it because otherwise it was getting the wrong (old) version.